torch/nn/functional.py

```diff
@@ -1153,8 +1153,16 @@ def batch_norm(input, running_mean, running_var, weight=None, bias=None,
     )
 
 
+def local_response_norm(input, size, alpha=1e-4, beta=0.75, k=1):
+    div = avg_pool3d(input.pow(2).unsqueeze(1), (size, 1, 1),
+                     stride=1, padding=(size // 2, 0, 0)).squeeze(1)
+    div.mul_(alpha).add_(k).pow_(beta)
```

Comment:
I don't think you want to `mul_` and `pow_` here (it's going to be inefficient)

Reply:
Removed inplace ops - is that what you meant?